src/Docker/Compose/ProductionCompose.php

```diff
@@ -31,6 +33,113 @@ class ProductionCompose implements ComposeInterface
 
     const CRON_ENABLED = true;
 
+    /**
+     * Extensions which should be installed by default
+     */
+    const DEFAULT_PHP_EXTENSIONS = [
```

Comment:
Why you didn't create a separate class for php extension-related logic? 
Something like `Docker\Config\PhpExtensions`.
`ProductionCompose` class became very large and has too much responsibility.

Reply:
Thank you!
Yes, you are right, it will be better